lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_check_beta_deps.rb

```diff
@@ -1,25 +1,50 @@
+require 'yaml'
+
 module Fastlane
   module Actions
     class IosCheckBetaDepsAction < Action
+      ALL_PODS_STABLE_MESSAGE = 'All pods are pointing to a stable version. You can continue with the code freeze.'.freeze
+      NON_STABLE_PODS_MESSAGE = "Please create a new stable version of those pods and update the Podfile to the newly released version before continuing with the code freeze:\n".freeze
```

Comment:
Nitpick. These messages assume the action runs as part of the code freeze. That is true in practice, and the action name sort of suggests it, too. Still, I'd suggest removing the code freeze mention to make it agnostic.